### DISCUSSION TITLE
Is it compatible with golang protobuf?

### DISCUSSION THREAD

**Discussion by zhengchengbin610** [protobuf-c/protobuf-c#619] [Q&A]
when  I  use protobuf-c  packed buf , and send it to a golang server , when the go decode the data appear this error:  proto: registerserver.Registermessage: illegal tag 0 (wire type 0) 
I  want  someone can help me , thanks very much!

I use protobuf-c 1.3.1 and  libprotoc 3.6.0
my proto file:
syntax="proto2";

message registermessage {

	required int32 type=1;
	required string version=3;
	required string wanip = 5;
	required int32 extport = 7;
	optional string lanip = 9;
	optional int32 lanport = 11;
}

**Comment by lipnitsk** [comment]
The encoded wire format is platform independent, that is a core purpose of protocol buffers. Please debug your implementation or the decoder.
